node/parachain/Cargo.toml

```diff
@@ -27,7 +27,7 @@ jsonrpc-core = "14.0.3"
 jsonrpc-pubsub = "14.0.3"
 
 # Parachain dependencies
-moonbase-runtime = { path = "../../runtime/parachain" }
+moonbeam-runtime = { path = "../../runtime" }
```

Comment:
So we used moonbase to differentiate from moonbeam. 
Moonbase is the name for the code/binary while moonbeam is for the final network (on Polkadot)

Reply:
This might change if you think we can have all our networks (rococo, kusama, polkadot...) using the same runtime.

Reply:
Oh yeah, I forgot we would have all those different deployments. We definitely can have them all use the same code. Whether it is a good idea probably depends how much we plan to have them diverge. If they are the same thing, or nearly the same thing, then I think it makes sense to use a single code base. I could add more features for pallets that are only in one of the deployments. But if we start to have lots of different pallets, it probably makes sense to split them back out.

In any case, I think it makes sense to merge them link this for now because:
1. We only have the two deployments for now
1. Less duplication is generally good
1. I did correct a few unintentional inconsistencies between the two runtimes (eg `CheckTxVersion`)

I don't have a strong feeling about what name we go with.

Reply:
Yes, I agree, my concern was about the naming of the runtime. (Having multiple runtime on different relaychain should appear as different names in the explorers)

When you said you were going to merge the runtimes, I thought we would have a common code base but still 2 libs.